dev/06_data_transforms.ipynb

```diff
@@ -73,7 +73,7 @@
     {
```

Comment:
<p>Note: we don't even need the test <code>len(folders)!=0</code> since folders is an L. Will remove it in a follow-up commit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Ah no, we miss the private folders in that case, forget I say antyhing .